src/controllers/ArticleController.ts

```diff
@@ -29,7 +31,51 @@ export class ArticleController extends AbstractController {
                 return res.status(200).json(answer)
             }
         });
+    }
+
+    public filterArticleByTitle = async(req:any, res:any) => {
+        let partialTitleName = req.params["_partialTitleName"]
+        if (!partialTitleName) {
```

Comment:
Unnecessary if statement

Reply:
you are right, if the title parameter were not in route that function would not be invoked.